src/body.js

```diff
@@ -206,7 +206,7 @@ function consumeBody() {
 
 	// body is buffer
 	if (Object.prototype.toString.call(this.body) === '[object ArrayBuffer]') {
-		return Body.Promise.resolve(this.body);
+		return Body.Promise.resolve(new Buffer(this.body));
```

Comment:
note that `new Buffer` is deprecated, use `Buffer.from` instead, see https://nodejs.org/api/buffer.html#buffer_new_buffer_buffer

This creates a copy of `this.body` as far as I know I’m not sure if that’s what we want?

Reply:
Thanks for taking a look. 

The docs say `Buffer.from()` "creates a view of the ArrayBuffer without copying the underlying memory." I think we can then decode (in `.text()` for example) without any additional memory overhead.